tests/lwip/Makefile

```diff
@@ -37,7 +37,11 @@ ifneq (,$(filter native,$(BOARD)))
 endif
 
 ifneq (,$(filter at86rf2%,$(DRIVER)))
-  FEATURES_REQUIRED = periph_spi periph_gpio
+  FEATURES_REQUIRED += periph_gpio
+endif
+
+ifeq (,$(filter native,$(BOARD)))
+  FEATURES_REQUIRED += periph_spi
 endif
```

Comment:
The `ifeq` block sets the required feature `periph_spi` if the `BOARD` is not `native`,
so if we build for a real hardware, including boards using an `at86rf2xx` driver, we require SPI support.
I thought, that was what @miri64 requested.